src/application/components/Home/store.js

```diff
@@ -1,5 +1,28 @@
 import Store from "../../common/StoreClass";
+import { get, save } from "../../common/api";
 
-class HomeStore extends Store {}
+class HomeStore extends Store {
+  async loadHomePageClicksCount() {
```

Comment:
store will contains actions all time or this is a temporary solution?

Reply:
why not?

Reply:
for parse data we can create some kind of model, need to think about this